src/control/controlobject.cpp

```diff
@@ -75,12 +79,100 @@ double ControlObject::getMidiParameter() const {
     return m_pControl->getMidiParameter();
 }
 
-// static
 double ControlObject::get(const ConfigKey& key) {
     QSharedPointer<ControlDoublePrivate> pCop = ControlDoublePrivate::getControl(key);
     return pCop ? pCop->get() : 0.0;
 }
 
+QString ControlObject::name() const {
+    return m_pControl ? m_pControl->name() : QString();
+}
+
+void ControlObject::setName(const QString& name) {
+    if (m_pControl) {
+        m_pControl->setName(name);
+    }
+}
+
```

Comment:
there was no reason to remove these from the header. please move all of these simple functions back to it so the compiler can inline them.